app/src/main/java/ch/epfl/qedit/model/User.java

```diff
@@ -24,13 +19,26 @@
 
     private final String firstName;
     private final String lastName;
-    private final Role role;
 
-    public User(String firstName, String lastName, Role role) {
+    private int score;
+    private int success;
```

Comment:
I feel like we could find more explicit names instead of `success` and `attempt`. Like `numSuccesses` and `numAttempts`, or `successes` and `attempts`...

Reply:
Otherwise, LGTM. It was nice to remove the roles, since we don't use them anymore !

Reply:
Yes right, my bad, it's my english ^^

Reply:
No problem ;).